scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -509,13 +505,14 @@
                     :namespace "cook"}
          scan-frequency-seconds 120
          max-pods-per-node 32
-         node-blocklist-labels (list)}}
+         node-blocklist-labels (list)
+         use-google-service-account? true}}
```

Comment:
Can we make these be lexicographically sorted?

Reply:
updated